timeless/restaurants/tables/views.py

```diff
@@ -0,0 +1,63 @@
+"""tables views module.
+@todo #112:30min Continue implementing list_tables(), create(), edit() and
+ delete() methods, using SQLAlchemy and Location model. In the index page it
+ should be possible to sort and filter for every column. Location management
+ page should be accessed by the Location page. Update html templates when
+ methods are implemented. Create more tests for all methods.
+ Table management pages should be accessed by the Location and Floor pages.
+"""
+import datetime
+
+from flask import (
+    Blueprint, flash, redirect, render_template, request, url_for
+)
+
+
+bp = Blueprint("table", __name__, url_prefix="/tables")
+
+
+@bp.route("/")
+def list():
+    # remove this dummy tables object and use db
+    tables = [{
+        "id": 1,
+        "name": "Test table",
+        "description": "This is a test",
+        "floor_id": 1,
+        "x": 10,
+        "y": 20,
+        "width": 12,
+        "height": 22,
+        "status": 1,
+        "max_capacity": 6,
+        "multiple": True,
+        "playstation": False,
+        "shape_id": 3,
+        "min_capacity": 4,
+        "deposit_hour": 3,
+        "created": datetime.datetime.now(),
+        "updated": datetime.datetime.now(),
+        }]
+    return render_template("restaurants/tables/list.html", tables=tables)
+
+
+@bp.route("/create", methods=("GET", "POST"))
+def create():
+    if request.method == "POST":
+        flash("Create not yet implemented")
+    action = "create"
+    return render_template("restaurants/tables/create_edit.html", action=action)
+
+
+@bp.route("/edit/<int:id>", methods=("GET", "POST"))
+def edit(id):
+    if request.method == "POST":
+        flash("Edit not yet implemented")
+    action = "edit"
+    return render_template("restaurants/tables/create_edit.html", action=action)
+
+
+@bp.route("/delete", methods=["POST"])
+def delete():
```

Comment:
@krzyk could we put these procedures into a class?

Reply:
@amihaiemil OK, but why? :)
All other views (except companies) are not in class. Is there a benefit to that?

Reply:
@amihaiemil well, if you don't have state then I don't see a point.
Maybe for json we will have another annotation and it won't bee needed also.

I followed the current desing in application, so it depends on @emilianodellacasa if he wants to change that, but this change would be needed for all views so it would be consistent

Reply:
@krzyk let's keep as it is now